core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1531,21 +1547,19 @@ public static boolean isClassProperty(String key) {
     propertiesByKey = new HashMap<>();
     validPrefixes = new HashSet<>();
     validProperties = new HashSet<>();
+    validTableProperties = new HashSet<>();
 
     for (Property p : Property.values()) {
+      propertiesByKey.put(p.getKey(), p);
```

Comment:
I can't tell if this got moved for aesthetics, or some other reason. Does it behave the same here as it did before, when it was below the if/else?

Reply:
Yes - I believe that it was always being assigned - so I moved it to the top so it was easier to see that was happening.